packages/bierzo-wallet/src/routes/welcome/index.dom.spec.ts

```diff
@@ -0,0 +1,34 @@
+import TestUtils from 'react-dom/test-utils';
+import { Store } from 'redux';
+import { configureStore } from '../../store';
+import { RootState } from '../../store/reducers';
+import { expectRoute } from '../../utils/test/dom';
+import { PAYMENT_ROUTE } from '../paths';
+import { travelToWelcome } from './test/travelToWelcome';
+
+describe('The /welcome route', () => {
+  let store: Store<RootState>;
+  let welcomeDom: React.Component;
+
+  beforeEach(
+    async (): Promise<void> => {
```

Comment:
Small side note: we changed the linter settings, such that an explicit return type is not necessary, **if the function is unnamed**. This is especially useful in test code:

```ts
beforeEach(async () => {
  // lalalalala
});
```

Reply:
Thanks! I will take it into account.